python/cuml/tests/test_umap.py

```diff
@@ -26,10 +26,11 @@
 import pytest
 import scipy.sparse as scipy_sparse
 import umap
+from cuvs.neighbors import all_neighbors, nn_descent
```

Comment:
We should not directly import the cuvs Python API in cuML. If we do, then we need to cuvs (not just libcuvs) to our test dependencies. CC @divyegala

Reply:
we can get rid of this, but if we do, we have to run the full e2e cpu umap on a not-so-small dataset for comparison (because outliers don't show up with small datasets)